cognitedata/reveal
# feat(react-components): Added support for point cloud annotation clickable and styling

#### Type of change

![Feat](https://img.shields.io/badge/Type-Feat-green)  

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/BND3D-3787

## Description :pencil:

Enabled point cloud annotation/asset selection using mouse click and enabled styling for point cloud models

## How has this been tested? :mag:

In Storybook


## Test instructions :information_source:

- 'yarn && yarn build && yarn storybook`
- Run ` HighlightNode` example
- Select any `blue` objects in `point-cloud` model which should highlight the object (points)


## Checklist :ballot_box_with_check:
